docs/configuration.md

```diff
@@ -1,228 +1,3 @@
-redirects or if [setting `old_paths` in the frontmatter](docs/frontmatter.md#old_paths) doesn't work.
-
-```yaml
-redirects:
-  /old-page.html: https://example.org/something-else.html
-  /another/old-page.html: /another/new-page.html
-```
-
-## `service_name`
-
-The service name in the header.
-
-Example:
-
-```yaml
-service_name: "Platform as a Service"
-```
-
-## `full_service_name`
-
-The full service name (maybe with GOV.UK)
-
-Example:
-
-```yaml
-full_service_name: "GOV.UK Pay"
-```
-
-## `service_link`
-
-What the service name in the header links to.
-
-default: '/'
-
-```yaml
-service_link: "/"
-```
-
-## `show_contribution_banner`
-
-Show a block at the bottom of the page that links to the page source, so readers
-can easily contribute back to the documentation. If turned on [github_repo](#github-repo) is
-required.
-
-Off by default.
-
-```yaml
-show_contribution_banner: true
-github_repo: alphagov/example-repo
-```
-
-## `source_urls`
-
-Customise the URLs that the contribution banner links to. Only useful if
-[show_contribution_banner](#show_contribution_banner) is turned on. By default, "Report issue" links
-to raising a GitHub issue but by modifying the `report_issue_url` it can link to an email address
-or another page.
-
-```yaml
-source_urls:
-  report_issue_url: mailto:support@example.com
-```
-
-## `show_govuk_logo`
-
-Whether to show the GOV.UK crown logo.
-
-default: `true`
-
-```yaml
-show_govuk_logo: true
-```
-
-## `api_path`
-
-Define a path to an Open API V3 spec file. This can be a relative file path or a URI to a raw file.
-
-```yaml
-api_path: ./source/pets.yml
-```
-
-## `owner_slack_workspace` and `default_owner_slack`
-
-These attributes are used to specify the owner of a page.  See the separate
-[documentation for page expiry][expiry] for more details.
-
-## `show_expiry`
-
-Decides whether or not to show a red banner when the page needs to be reviewed.
-
-If not present or set to `true`, the red banner will appear when the page needs to be reviewed. This is the default behaviour.
-
-If set to `false`, the red banner will not appear when the page needs to be reviewed.
-
-See the separate [documentation for page expiry][expiry] for more details.
-
-[expiry]: https://tdt-documentation.london.cloudapps.digital/page-expiry.html#page-expiry-and-review
-
-## `show_review_banner`
-
-Decides whether or not to display the page review banner, regardless of whether the page needs to be reviewed or not.
-
-If not present or set to `true`, the banner will be displayed on the page. This is the default behaviour.
-
-If set to `false`, the banner will not be displayed.
-
-See the separate [documentation for page expiry][expiry] for more details.
-
-[expiry]: https://tdt-documentation.london.cloudapps.digital/page-expiry.html#page-expiry-and-review
+Content has been moved to alphagov/tdt-documentation: https://github.com/alphagov/tdt-documentation/pull/108/commits/e50c6bc0dd43b2f299444b7dc760e1a85edcb718
```

Comment:
Would it be more useful to link the user to the live documentation site for these links (e.g. https://tdt-documentation.london.cloudapps.digital/amend_project/configuration/#configuration-options)?

Reply:
My reasoning for putting that link in was to explain why it was moved. But I think you're right, it's probably more helpful to tell users where the content was moved to.

How about changing it to something like:

This content has been moved to: https://tdt-documentation.london.cloudapps.digital/amend_project/configuration/#configuration-options

So you think it's worth linking to the source file too?

Reply:
Added links to where the content is published: https://github.com/alphagov/tech-docs-gem/pull/151/commits/244142097cc6f5b85ba4ab5c6f0674dc2c690c53

The pages have links to the `alphagov/tdt-documentation` repo as well as the contribution banner so it should be fairly straightforward to find the source.